ospf6d/ospf6_intra.c

```diff
@@ -2111,7 +2111,7 @@ static void ospf6_brouter_debug_print(struct ospf6_route *brouter)
 	char installed[64], changed[64];
 	struct timeval now, res;
 	char id[16], adv_router[16];
-	char capa[16], options[16];
+	char capa[16], options[32];
```

Comment:
Why is the size change `16` → `32` needed here?  Should the `32` be replaced with a constant?

Reply:
Its just used to convert the options in ospf6 header. with addition of LLS and Authentication trailer the size extended so need to increase the size of the options string.

Reply:
Since it is used in a lot of places, can you please change it to a `#define`?

Reply:
Since there are so many constants along with it, as part of options derivation. I feel let's keep it as it is for now. We can take it up as separate PR. What do you think ?